media-proxy/src/api_server_tcp.cc

```diff
@@ -212,16 +160,8 @@ void* msg_loop(void* ptr)
                 break;
             }
             session_id = *(uint32_t*)buffer;
-            for (auto it : proxy_ctx->mDpCtx) {
-                if (it->id == session_id) {
-                    if (it->type == TX) {
-                        proxy_ctx->TxStop(session_id);
-                    } else {
-                        proxy_ctx->RxStop(session_id);
-                    }
-                    sessionKeepRunning = false;
-                    break;
-                }
+            if(!proxy_ctx->Stop(session_id)){
```

Comment:
You hate spaces, don't you? ;-)
```suggestion
            if (!proxy_ctx->Stop(session_id)) {
```

Reply:
I recommend using: `git-clang-format main`

the `.clang-format` file is configured to format everything approximately according to our codestyle.

Reply:
I did it for all new files. Applying clang-format for existing files can show much more changes and reduce readability

Reply:
you can use clang-format not only to format files but also to format commits. `git-clang-format main` formats everything changed in comparison to the `main` branch.

Reply:
done

Reply:
doesn't look like done

Reply:
fix in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/240/commits/36026af73506c0b23e4928f80afa3aaebd361e8f